include/seqan3/core/algorithm/detail/blocking_algorithm_executor.hpp

```diff
@@ -34,7 +34,7 @@ namespace seqan3::detail
  * \ingroup execution
  * \tparam resource_t            The underlying range of sequence pairs to be computed; must model
  *                               std::ranges::viewable_range and std::ranges::input_range.
- * \tparam alignment_algorithm_t The alignment algorithm to be invoked on each sequence pair.
+ * \tparam algorithm_t The algorithm to be invoked on the elements of the given resource; must model std::semiregular.
  * \tparam value_t               The value type to buffer results generated from the algorithm; must model
```

Comment:
```suggestion
 * \tparam algorithm_t           The algorithm to be invoked on the elements of the given resource; must model std::semiregular.
 * \tparam value_t               The value type to buffer results generated from the algorithm; must model
```

Reply:
well, it is the otherway around. Thanks for spotting.

Reply:
this is already fixed in some commit.